hive-metastore/src/main/java/org/apache/iceberg/hive/HiveHadoopUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.hive;
+
+import java.io.IOException;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HiveHadoopUtil {
+
+  private static final Logger LOG = LoggerFactory.getLogger(HiveHadoopUtil.class);
+
+  private HiveHadoopUtil() {}
+
+  public static String currentUser() {
+    try {
+      return UserGroupInformation.getCurrentUser().getUserName();
+    } catch (IOException e) {
+      LOG.warn("Fail to obtain Hadoop UGI user", e);
```

Comment:
Minor: to fit with conventions, I think the error message should be `Failed to get Hadoop user`.
* Error messages use past tense to describe what was attempted
* "Obtain" is no more specific than "get", so prefer the simpler word
* `UGI` is an internal Hadoop term that is more confusing than helpful

Reply:
Adopted in the latest commit.